docs/index.md

```diff
@@ -56,3 +56,16 @@ patterns from Cirrus CI customers:
     <td></td>
   </tr>
 </table>
+
+<script type="text/javascript">
```

Comment:
I'm hesitant to add such script especially since the documentation is under MIT.

Plus I like the rounded number and the exact number will look odd.

Reply:
@paramt you comfortable removing the copyright statement?

I can add something to round it if you really want

Reply:
Nah. I think after 100,000 we can stop updating it. 😆

Reply:
I would say a million would also need an update though...

Reply:
@RDIL Go ahead 👍

Reply:
[Another personal project](https://github.com/MasMas-js/MasMas.js/blob/master/masmas.js) has both commafy and rounding in it